deployment/grid/terraform/compute_plane/eks_cluster.tf

```diff
@@ -241,6 +232,45 @@ module "eks_blueprints_kubernetes_addons" {
     })]
   }
 
+#  enable_aws_cloudwatch_metrics = true
+#  aws_cloudwatch_metrics_helm_config = {
+#    values = [
+#      templatefile("${path.module}/../../charts/values/aws-cloudwatch-metrics.yaml", {
+#        aws_htc_ecr    = var.aws_htc_ecr
+#        eks_cluster_id = module.eks.eks_cluster_id
+#      })
+#    ]
+#  }
+
+  depends_on = [
+    null_resource.patch_coredns
+  ]
+}
+
+module "eks_blueprints_kubernetes_addons_for_alb" {
```

Comment:
Didn't we say we would just move to the latest version of the blueprints repo and then keep it as it was/the same across all the addons?